spec/factories/instructors.rb

```diff
@@ -6,5 +6,8 @@
     password { Faker::Internet.password }
     status { :active }
     role { :instructor }
+    after(:build) do |instructor|
+      instructor.avatar.attach(io: File.open(Rails.root.join('spec', 'factories', 'images', 'default_profile.png')), filename: 'default_profile.png', content_type: 'image/png')
```

Comment:
`spec/factories/images/default_profile.png` has 139 KB. Please for specs use small image (less than 1 KB)